connectors/api/Makefile

```diff
@@ -0,0 +1,18 @@
+ROOT_DIR := ../..
```

Comment:
Why is this Makefile required? I don't see any usage for "generate-client-datacatalog" target. Besides, the open-api client for datacatalog has already been generated in the master version without using this rule, so what is changed now?

Reply:
yes I have just added this temporarily just to show how the openapi client was re-generated. The change to the open api client is that there is additional code generated in api_default.go related to createAssetInfo. I will remove it anyways. Thanks.

Reply:
I suggest you document this and not in the Makefile, for example in README or perhaps there is a more appropriate place.

Reply:
Ok let me think about this. Please suggest if there is a good candidate for this. It may help in future also if someone needs to add a new api.

Reply:
I think for now I will not include the Makefile details as the creatAssetInfo method is still under development. Once we are able to complete the end to end testing (which may take about 2-3 PRs more) then we can decide a way to document this. Hope this is ok. Thanks.